include/nbl/video/IPhysicalDevice.h

```diff
@@ -26,75 +26,167 @@ class IPhysicalDevice : public core::Interface, public core::Unmovable
     public:
         //
         virtual E_API_TYPE getAPIType() const = 0;
-        
-        // TODO: fold into SLimits
+
+        enum E_TYPE : uint8_t {
+            ET_OTHER = 0,
+            ET_INTEGRATED_GPU = 1,
+            ET_DISCRETE_GPU = 2,
+            ET_VIRTUAL_GPU = 3,
+            ET_CPU = 4,
+        };
+
+        enum E_DRIVER_ID : uint8_t
+        {
+            EDI_OTHER = 0,
```

Comment:
UNKNOWN

Reply:
"Other" here has the same name as the Vulkan enum. It kinda makes sense. But for consistently I will make it UNKNOWN